packages/markers/README.md

```diff
@@ -0,0 +1,6 @@
+# Theia - Markers Extension
+
+See [here](https://github.com/theia-ide/theia) for a detailed documentation.
```

Comment:
We still could write a brief description here ? The main theia webpage doesn't have much doc about it.